src/ArtificialViscosity/ArtificialViscosity.hh

```diff
@@ -180,12 +180,12 @@ public:
   virtual void restoreState(const FileIO& file, const std::string& pathName);
   //****************************************************************************
 
-protected:
-  //--------------------------- Protected Interface ---------------------------//
   // Allow descendents to request that sigma be calculated.
   bool calculateSigma() const;
   void calculateSigma(bool value);
 
+protected:
```

Comment:
Why was it necessary to move the sigma functions to public from protected?

Reply:
It was previously, let me check with the latest library changes since we are no longer building the python libs with nvcc.

Reply:
Reverted this change.